nodejs/eks/cluster.ts

```diff
@@ -313,8 +313,8 @@ export function getRoleProvider(
     parent?: pulumi.ComponentResource,
     provider?: pulumi.ProviderResource,
 ): CreationRoleProvider {
-    const partition = pulumi.output(aws.getPartition({ parent })).partition;
-    const accountId = pulumi.output(aws.getCallerIdentity({ parent })).accountId;
+    const partition = pulumi.output(aws.getPartition({})).partition;
```

Comment:
Can we just use [`aws.getPartitionOutput`](https://www.pulumi.com/registry/packages/aws/api-docs/getpartition/) here?
```suggestion
    const partition = aws.getPartitionOutput({ parent }).partition;
```